google-chart.ts

```diff
@@ -489,8 +489,7 @@ export class GoogleChart extends LitElement {
     const {rows, cols} = this;
     if (!rows || !cols) return;
```

Comment:
Should this set this._data to undefined?

Reply:
Do you mean whether to do that in order to clear the chart by setting `undefined` rows or columns? Given that other input variants (datatable and view) also skip `undefined` without clearing the chart, I'd keep it as is.